app/models/game.rb

```diff
@@ -36,4 +36,27 @@ def other_player(player)
   def pass_turn(player)
     update_attributes(active_player_id: other_player(player))
   end
+
+  def stalemate?
+    king_not_in_check? && available_move?
```

Comment:
should this actually be `king_not_in_check? && !available_move?` or `king_not_in_check? && no_available_move?` or something?